includes/abstract-class-taxjar-record.php

```diff
@@ -79,14 +81,22 @@ public function create() {
 
 		global $wpdb;
 		$insert = array(
-			'record_id'        => $this->get_record_id(),
-			'record_type'      => $this->get_record_type(),
-			'status'           => $this->get_status(),
-			'batch_id'         => $this->get_batch_id(),
-			'created_datetime' => $this->get_created_datetime(),
-			'force_push'       => $this->get_force_push()
+			'record_id'          => $this->get_record_id(),
+			'record_type'        => $this->get_record_type(),
+			'status'             => $this->get_status(),
+			'batch_id'           => $this->get_batch_id(),
+			'created_datetime'   => $this->get_created_datetime(),
+			'force_push'         => $this->get_force_push()
 		);
 
+		if ( ! empty( $this->get_processed_datetime() ) ) {
+			$insert[ 'processed_datetime' ] = $this->get_processed_datetime();
+		}
+
+		if ( $this->get_last_error() === "" | ! empty( $this->get_last_error() ) ) {
```

Comment:
@dallendalton Should this be `||` instead of `|`?

Reply:
Yeah it should have.... turns out php also accepts the single | operator with no issues, though I don't see it documented. Which is why none of the specs or my manual testing had issues.